pypeit/scripts/chk_for_calibs.py

```diff
@@ -137,15 +139,17 @@ def main(args):
             answers['scifiles'][i] = ''
 
         # Check!
-        answers['pass'][i] = calibrations.check_for_calibs(par, ps.fitstbl, raise_error=False,
-                                                           cut_cfg=in_cfg)
+        answers['pass'][i] = calibrations.check_for_calibs(par, ps.fitstbl,
+                                                           raise_error=False, cut_cfg=in_cfg)
         if not answers['pass'][i]:
             msgs.warn("Setup {} did not pass the calibration check!".format(setup))
 
     print('= RESULTS ============================================')
     # Print
     answers.pprint_all()
     print('======================================================')
+    # Remove setup_files
+    shutil.rmtree('setup_files')
```

Comment:
Do you want to always remove the setup files or would it make sense to make it a command-line option to keep them?

Reply:
Good suggestion.  Added.